aptos-move/mvhashmap/src/unit_tests/proptest_types.rs

```diff
@@ -211,7 +211,7 @@ where
 
     let baseline = Baseline::new(transactions.as_slice(), test_group);
     // Only testing data, provide executable type ().
-    let map = MVHashMap::<KeyType<K>, usize, Value<V>, ExecutableTestType>::new();
+    let map = MVHashMap::<KeyType<K>, usize, usize, Value<V>, ExecutableTestType>::new();
```

Comment:
Why is identifier type usize here instead of u64?

Reply:
Same as above